packages.yaml

```diff
@@ -1,3 +1,4 @@
 - source: https://github.com/anexia-it/go-anxcloud.git
 - source: https://github.com/anexia-it/go-cloudlog.git
-- source: https://github.com/anexia-it/go-e5e.git
+- source: https://github.com/anexia/go-e5e.git
+  summary: Support library to help Go developers build Anexia e5e functions
```

Comment:
if you move your CI & co badges above the first header, it should take the summary from the README.md :)

Reply:
As go.anx.io takes the summary from the first header, I'd rather stick to the summary defined in the `packages.yaml`.

Reply:
oki, just wanted to point out that option :)